src/utils/file-upload-utils.js

```diff
@@ -11,10 +14,28 @@ const ALLOWED_FILE_EXTENSIONS = [
   "bmp",
   "ico",
 ]
+const defaultClient = CloudmersiveVirusApiClient.ApiClient.instance;
 
-const validateAndSanitizeFileUpload = async (data) => {
-  const [schema, content] = data.split(",")
-  const fileBuffer = Buffer.from(content, "base64")
+// Configure API key authorization: Apikey
+var Apikey = defaultClient.authentications['Apikey']
```

Comment:
`var` should **never** be used because it's in the global scope :monkas: (see [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/var) for details)

also, since `ApiKey` seems to be an instance, could we have it in `camelCase`? (see [here](https://www.robinwieruch.de/javascript-naming-conventions/) for casing conventions)

Reply:
You are absolutely right. I took the sample code from [CloudMersive API reference](https://api.cloudmersive.com/docs/virus.asp) and forgot to change this line before committing. Also changed `authentications['Apikey']` to a dot reference `authentications.Apikey` 
Fixed in 6ebc12a9cfe977f4a309329fc571ae04322bb93c